ethcore/src/blockchain/best_block.rs

```diff
@@ -19,7 +19,8 @@ use bigint::hash::H256;
 use bytes::Bytes;
 use header::BlockNumber;
 
-/// Best block info.
+/// Best block info. Best block is the block with highest combined difficulty
+/// (usually the block with the highest block number). Sometimes refered as latest block.
```

Comment:
best block heuristic is really dependent on the consensus engine. this new description is only applicable to the GHOST fork-choice rule

Reply:
Should I rollback the comment entirely, or provide some extra text like this: 
```Rust
/// Contains information on a best block that is specific to the consensus engine. 
///
/// For GHOST fork-choice rule it would typically describe the block with highest 
/// combined difficulty (usually the block with the highest block number).
/// 
/// Sometimes refered as 'latest block'.
```
?

Reply:
text like that is fine. a couple nits, though: `combined` -> `cumulative` and the last line about sometimes being referred to as the latest block is correct, but those would be misuses of the term, so I'd recommend omitting that statement.